0.preprocess-sites/4.image-and-segmentation-qc.py

```diff
@@ -504,14 +509,47 @@
         )
 
     image_corr_df = (
-        pd.concat(image_corr_list).drop_duplicates(subset="site_location").reset_index()
+        pd.concat(image_corr_list).drop_duplicates(subset="site").reset_index()
```

Comment:
```suggestion
        pd.concat(image_corr_list).drop_duplicates(subset="site_location").reset_index()
```

Is this right?

Reply:
No, I think it's correct with `site` not `site_location`.